api/api_endpoints.py

```diff
@@ -0,0 +1,72 @@
+from flask import Blueprint, jsonify, request
```

Comment:
the module name  has duplicate wording in it
`api/api_endpoints.py`

it would be much readable if we had it to read
`api/endpoints.py`

Reply:
Yeah, that makes sense.